docs/content/guide/migration.ngdoc

```diff
@@ -13,6 +13,156 @@ which drives many of these changes.
 * Several new features, especially animations, would not be possible without a few changes.
 * Finally, some outstanding bugs were best fixed by changing an existing API.
 
+
+
+
+
+# Migrate from 1.4 to 1.5
+
+Angular 1.5 takes a big step towards enabling developers to prepare themselves for a smoother
```

Comment:
could be shortened to  `.. a big step towards preparing developers for a smoother ..`